### DISCUSSION TITLE
trame-master can't display things，file is XMLPolyData(vtp)

### DISCUSSION THREAD

**Discussion by 865979216** [Kitware/trame#389] [Q&A]
trame-master\examples\06_vtk\Applications\VTPViewer\app.py
When I run it, I choose output.vtp。
![image](https://github.com/Kitware/trame/assets/97671318/fbaeaa86-7050-44b6-b077-c297bb3659ba)
but ，nothing 
![image](https://github.com/Kitware/trame/assets/97671318/de744af4-e979-4e69-985f-8f857dc7449c)
Please help me，I need your help！！

**Comment by jourdain** [comment]
Can you share your VTP file? Maybe the example is not meant to work with your VTP. 
It seems to be working when using the vtp file inside `trame/examples/data`.

<img width="1272" alt="Screenshot 2023-12-11 at 09 14 17" src="https://github.com/Kitware/trame/assets/294518/da11618b-ea4a-499d-b9dd-d24742115c0b">

**Reply by 865979216** [reply]
我需要刷新网页，然后它可以显示
f1.vtp 也需要刷新。

**Comment by 865979216** [comment]
I need refresh web, then it can show
![image](https://github.com/Kitware/trame/assets/97671318/8afee83a-46cb-46dd-95ea-5f7da683dec0)
f1.vtp is also need to refresh.
